### DISCUSSION TITLE
请问在1.45k步数下，未出现注意力模型，但是loss已经低于0.3，是否说明样本太少？

### DISCUSSION THREAD

**Discussion by fatinghenji** [babysor/MockingBird#864] [Q&A]
RT，补充图像如下：
![attention_step_14500_sample_1](https://user-images.githubusercontent.com/37131177/228794379-6caa20c9-792f-4ef6-8c44-273c1d1a30be.png)
![step-14500-mel-spectrogram_sample_1](https://user-images.githubusercontent.com/37131177/228794388-e221e379-208a-4f24-ab81-c2730aa63b62.png)
总样本库约1.5小时，从头开始训练。

**Comment by YIZXIY** [comment]
放弃吧，我记得作者在之前的回答里说至少要5个G还是说要100个小时来着，所以除非音源是天天演讲的知名人物，但是这样的人是不可能会让你练模型的，拿那个75K的模型做一些增量训练效果会非常好的
